### DISCUSSION TITLE
Error during make install - cannot stat './lotus': No such file or directory - Raspberry pi 4

### DISCUSSION THREAD

**Discussion by ironking63** [filecoin-project/lotus#7142] [General Q&A]
Greeting,
In my last step when I try "sudo make install" I face the error below:
bash: go: command not found
expr: syntax error: unexpected argument '1016000'
install -C ./lotus /usr/local/bin/lotus
install: cannot stat './lotus': No such file or directory
make: *** [Makefile:128: install-daemon] Error 1

I use Raspbian latest version.
What is this error for?

**Comment by de-authority** [comment]
Do you compile successed?
it's usually happen when you didn't complete the compile or you move the binary file after compile.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
> bash: go: command not found

export the path to your go binaries

**Comment by LexLuthr** [comment]
@ironking63 Were you able to solve this compilation problem? if yes, can you let us know how you fixed it?
